BEProjectsApp/models.py

```diff
@@ -1,3 +1,81 @@
 from django.db import models
+from django.utils import timezone
 
-# Create your models here.
+
+DOMAIN_CHOICES = [
+    (1, ("none")),
+    (2, ("Data Mining & Analytics")),
+    (3, ("Machine Learning")),
+    (4, ("Deep Learning")),
+    (5, ("Image Processing/Computer Vision")),
+    (6, ("Natural Language Processing/Artificial Intelligence")),
+    (7, ("Networking/Security")),
+    (8, ("Internet of Things(IOT)")),
+    (9, ("Mobile Computing")),
+    (10, ("Big Data")),
+    (11, ("Cloud Computing")),
+    (12, ("Computer Vision & Artificial Intelligence")),
+    (13, ("Blockchain")),
+]
+
+
+class Teacher(models.Model):
+
+    teacher_id = models.AutoField(primary_key=True)
+
+    teacher_name = models.CharField(max_length=150)
+
+    username = models.CharField(max_length=150)
+
+    password = models.CharField(max_length=256)
+
+    subject = models.CharField(max_length=150)
+
+    def __str__(self):
+
+        return self.teacher_name
+
+
+# In house Project model
+class IN_Project(models.Model):
```

Comment:
Can we give it a better name?

Reply:
In_House_Projects Or college projects for inhouse and company projects for outhouse ?

Reply:
or how is `inhouse_project`?

Reply:
ok done